From f4dac01833624f29c1588baa5da5d5dd809ae115 Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Tue, 13 Feb 2024 15:24:15 +0100
Subject: [PATCH 0931/1141] arm64: dts: st: Add st,rx_equalizer tuning on
 stm32mp257f-ev1

Tune the RX Equalizer on EV1 board. Best value "1" found with
Gen2 SATA benchmark.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: Iecc0fb27a232d1a03c445f3a1f1663c4f944ff0c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/359658
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp257f-ev1.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
index 11ecc9f69347..1af9740154b7 100644
--- a/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
+++ b/arch/arm64/boot/dts/st/stm32mp257f-ev1.dts
@@ -159,6 +159,7 @@ &arm_wdt {
 &combophy {
 	clocks = <&rcc CK_BUS_USB3PCIEPHY>, <&rcc CK_KER_USB3PCIEPHY>, <&pad_clk>;
 	clock-names = "apb-clk", "ker-clk", "pad-clk";
+	st,rx_equalizer = <1>;
 	status = "okay";
 };
 
-- 
2.39.2

